packages/dom-adapters/src/BlockToolAdapter/index.ts

```diff
@@ -351,6 +351,16 @@ export class BlockToolAdapter implements BlockToolAdapterInterface {
     const currentValue = this.#model.getText(this.#blockIndex, key);
     const newValueAfter = currentValue.slice(end);
 
+    const relatedFragments = this.#model.getFragments(this.#blockIndex, key, end, currentValue.length);
+
+    /**
+     * Adjust fragments ranges respectfully to the removed text
```

Comment:
I can't understand what does it mean "adjust". Could you write more detailed explanation please?